docs/self-managed/concepts/pvc-resize-guide.md

```diff
@@ -0,0 +1,163 @@
+# 🔁 PVC Expansion or Migration Guide
+## ✅ Approach 1: Direct PVC Expansion (if allowVolumeExpansion: true)
+
+### Step 0: Check if Your StorageClass Supports Expansion
+```bash
+kubectl get storageclass -o yaml | grep allowVolumeExpansion
+```
+
+Or for a specific class:
+
+```bash
+kubectl get storageclass <your-storage-class> -o yaml | grep -A5 -B5 allowVolumeExpansion
+```
+
+### Step 1: Check Current PVCs
+```bash
+kubectl get pvc -l app.kubernetes.io/component=data -o wide
+```
+
+### Step 2: Resize PVCs Directly
+```bash
+# Example: Resize two Elasticsearch data PVCs
+kubectl patch pvc data-elasticsearch-data-0 -p '{"spec":{"resources":{"requests":{"storage":"200Gi"}}}}'
+kubectl patch pvc data-elasticsearch-data-1 -p '{"spec":{"resources":{"requests":{"storage":"200Gi"}}}}'
+```
+
+
+#### Monitor:
+
+```bash
+kubectl get pvc -l app.kubernetes.io/component=data -w
+```
+
+### Step 3 (Optional): Update Helm Values
+```yaml
+# values.yaml for bitnami ES chart
+data:
+    persistence:
+        size: 200Gi
+```
+
+```bash
+helm upgrade elasticsearch oci://registry-1.docker.io/bitnamicharts/elasticsearch \
+--version 21.6.3 \
+-f values.yaml
```

Comment:
goes hand in hand with the error mentioned above.
You can not change it in the values.yml unless you remove the statefulset and let it be re-created.

Reply:
also tricky again since Camunda doesn't consume the artifact that way but the Camunda Helm Chart includes it.